<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerWindow</class>
 <widget class="QMainWindow" name="ManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>716</width>
    <height>505</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选课系统-管理员</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Client.qrc">
    <normaloff>:/Client/icon.png</normaloff>:/Client/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>课程</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0" colspan="6">
         <widget class="QTableWidget" name="classes">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>编号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>名称</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>类型</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>总学时</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>授课学时</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>实验学时</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>学分</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>开课学期</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>选择人数</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>169</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>169</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="removeClass">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="addClass">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="editClass">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="viewClass">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>详情</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>学生</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="6">
         <widget class="QTableWidget" name="students">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>学号</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>姓名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>选课数量</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>学分数</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>所选课程</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="1" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>169</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="addStudent">
          <property name="text">
           <string>添加</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <widget class="QPushButton" name="editStudent">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>编辑</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QPushButton" name="removeStudent">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>删除</string>
          </property>
         </widget>
        </item>
        <item row="1" column="4">
         <widget class="QPushButton" name="viewStudent">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>详情</string>
          </property>
         </widget>
        </item>
        <item row="1" column="5">
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>169</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>用户</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0" rowspan="6">
         <widget class="QTableWidget" name="users">
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="alternatingRowColors">
           <bool>true</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>用户名</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>用户类型</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="addUser">
          <property name="text">
           <string>添加用户</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QPushButton" name="removeUser">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>移除用户</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="resetUser">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>重置密码</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QPushButton" name="typeUser">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>权限更改</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QPushButton" name="regUser">
          <property name="text">
           <string>从学生注册</string>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>266</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>716</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>任务</string>
    </property>
    <addaction name="actionGet_Task"/>
    <addaction name="actionSet_Task"/>
    <addaction name="separator"/>
    <addaction name="actionNew_Task"/>
    <addaction name="actionSave_Task"/>
    <addaction name="actionLoad_Task"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Classes"/>
    <addaction name="actionSave_Classes"/>
    <addaction name="separator"/>
    <addaction name="actionLoad_Students"/>
    <addaction name="actionSave_Students"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>服务</string>
    </property>
    <addaction name="actionServer_Config"/>
    <addaction name="actionServer_Information"/>
    <addaction name="separator"/>
    <addaction name="actionServer_Shutdown"/>
    <addaction name="actionServer_Restart"/>
    <addaction name="separator"/>
    <addaction name="actionLog_Manage"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>账户</string>
    </property>
    <addaction name="actionChange_Password"/>
    <addaction name="actionLogout"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <action name="actionGet_Task">
   <property name="text">
    <string>拉取任务</string>
   </property>
  </action>
  <action name="actionSet_Task">
   <property name="text">
    <string>推送任务</string>
   </property>
  </action>
  <action name="actionNew_Task">
   <property name="text">
    <string>新建任务</string>
   </property>
  </action>
  <action name="actionSave_Task">
   <property name="text">
    <string>保存任务</string>
   </property>
  </action>
  <action name="actionLoad_Task">
   <property name="text">
    <string>打开任务</string>
   </property>
  </action>
  <action name="actionServer_Config">
   <property name="text">
    <string>服务器配置</string>
   </property>
  </action>
  <action name="actionServer_Information">
   <property name="text">
    <string>服务器信息</string>
   </property>
  </action>
  <action name="actionServer_Shutdown">
   <property name="text">
    <string>关闭服务端应用</string>
   </property>
  </action>
  <action name="actionServer_Restart">
   <property name="text">
    <string>重启服务端应用</string>
   </property>
  </action>
  <action name="actionChange_Password">
   <property name="text">
    <string>更改密码</string>
   </property>
  </action>
  <action name="actionLogout">
   <property name="text">
    <string>注销</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于系统</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>关于Qt</string>
   </property>
  </action>
  <action name="actionLog_Manage">
   <property name="text">
    <string>日志管理</string>
   </property>
  </action>
  <action name="actionLoad_Classes">
   <property name="text">
    <string>载入课程</string>
   </property>
  </action>
  <action name="actionSave_Classes">
   <property name="text">
    <string>保存课程</string>
   </property>
  </action>
  <action name="actionLoad_Students">
   <property name="text">
    <string>载入学生</string>
   </property>
  </action>
  <action name="actionSave_Students">
   <property name="text">
    <string>保存学生</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Client.qrc"/>
 </resources>
 <connections/>
</ui>
